jwcrypto/tests.py

```diff
@@ -695,27 +695,26 @@ def test_E(self):
             s.verify(None)
 
     def test_customhdr_jws(self):
+        # Test pass header check
         s = jws.JWS(A6_example['payload'])
```

Comment:
I would expect JWS to be initialized with the dictionary that defines a custom header check ?

Reply:
Got it, i added the header later on, but agree would be fitting to have it as a custom registry.